faq/messaging-and-queuing.mdx

```diff
@@ -23,6 +23,7 @@ We currently implement the API endpoints listed [here](/serverless/messaging/ref
 The following subscriber types are supported:
 - HTTP/S clients
 - Serverless Functions and Containers
+- Scaleway Messaging and Queuing SQS queues
```

Comment:
"Messaging and Queuing SQS"  only right ?
However, I like the fact that you mentioned the SQS resource.
So we stay with your proposition